### PR TITLE
Add Japanese localization

### PR DESCRIPTION
**Pull request by yuryleb**
# Issue

Initial Japanese translation contributed by @joshandrews (#276).
Thanks a lot!

## Tasklist

 - [x] Add changelog entry
 - [x] Test with osrm-frontend (?)
 - [x] Review

### CODE REVIEW

**Comment by yuryleb** [comment]
Unfortunately I could not check frontend in Japan (router.project-osrm.org always returned Route error for any requests inside Japan cities somehow), please look at Japanese instructions over US map:
Japanese-osrmti

File: languages/translations/ja.json [hunk 1]
```diff
@@ -0,0 +1,493 @@
+{
+    "meta": {
+        "capitalizeFirstLetter": false
+    },
+    "v5": {
+        "constants": {
+            "ordinalize": {
+                "1": "1 番",
```
Comment (friedbunny) [inline languages/translations/ja.json]: I’m not sure about the project-specific requirements here, but in general there would be no space between a number and a counting word (like 番).

Comment (yuryleb) [reply languages/translations/ja.json]: I think so. @joshandrews, could you please fix this in Transifex?

**Comment by joshandrews** [comment]
Thanks for the feedback everyone. A couple updates:

1. We're updating some of the Transifex translations based on this feedback and from viewing in the frontend.  **I'll update here when we've finished that**.  
2. We've created a PR to translate osrm-frontend found [here](https://github.com/Project-OSRM/osrm-frontend/pull/299)

**Comment by joshandrews** [comment]
We have updated the translations.  @yuryleb can you pull in most recent from Transifex?

File: languages/translations/ja.json [hunk 2]
```diff
@@ -0,0 +1,493 @@
+{
+    "meta": {
+        "capitalizeFirstLetter": false
+    },
+    "v5": {
+        "constants": {
+            "ordinalize": {
+                "1": "1 つ目の",
```
Comment (yuryleb) [inline languages/translations/ja.json]: There still are whitespaces after digits, is it ok?

Comment (joshandrews) [reply languages/translations/ja.json]: Yes this is intended.  This structure should work best based on the way sentences are formed.

**Comment by joshandrews** [comment]
@yuryleb Let me know if there is anything I can do to help get this past the finish line.

**Comment by yuryleb** [comment]
We did all we could, the PR could merged by maintainers only

**Comment by 1ec5** [comment]
Thanks for all your help @joshandrews @yuryleb!

**Pull request merged**